docs/collecting-data/collecting-from-own-applications/python-tracker/emitters/index.md

```diff
@@ -38,81 +38,51 @@ def __init__(
     )-> None:
 ```
 
-:::note Prior to v0.12.0
-
-Before version 0.12.0 the default values for the emitter `protocol` and `method` were `http` and `get` respectively. 
-
-:::
-:::note Prior to v0.13.0
-
-Before version 0.13.0 `batch_size` was named `buffer_size`
-
-:::
-| **Argument** | **Description** | **Required?** | **Type** | **Default** | **Version** | 
-| --- | --- | --- | --- | --- | --- |
-| `endpoint` | The collector URI | Yes | String | |
-| `protocol` | Request protocol: http or https | No | String | `https` | |
-| `port` | The port to connect to | No | Positive integer | `None` | |
-| `method` | The method to use: “get” or “post” | No | String | `post` | |
-| `batch_size` | Number of events to store before flushing | No | Positive integer | `10` | |
-| `on_success` | Callback executed when a flush is successful | No | Function taking 1 argument | `None` | |
+| **Argument** | **Description** | **Required?** | **Type** | **Default** |
+| --- | --- | --- | --- | --- | 
+| `endpoint` | The collector URI | Yes | String | 
+| `protocol` | Request protocol: http or https | No | String | `https` | 
+| `port` | The port to connect to | No | Positive integer | `None` | 
+| `method` | The method to use: “get” or “post” | No | String | `post` | 
+| `batch_size` | Number of events to store before flushing | No | Positive integer | `10` | 
+| `on_success` | Callback executed when a flush is successful | No | Function taking 1 argument | `None` | 
 | `on_failure` | Callback executed when a flush is unsuccessful | No | Function taking 2 arguments | `None` |
 | `byte_limit` | Number of bytes to store before flushing | No | Positive integer | `None` |
-| `request_timeout` | Timeout for HTTP requests | No | Positive integer or tuple of 2 integers | `None` | v0.10.0 |
-| `max_retry_delay_seconds` | The maximum time between attempts to send failed events to the collector | No | int | 60s | v0.13.0 |
-| `buffer_capacity` | The maximum capacity of the event buffer | No | int | `None` | v0.13.0 |
-| `custom_retry_codes` | Custom retry rules for HTTP status codes received in emit responses from the Collector | No | dict | `None` | v0.13.0 |
-| `event_store` | Stores the event buffer and buffer capacity | No | EventStore | `None` | v0.13.0 |
-| `session` | Persist parameters across requests by using a session object | No | requests.Session | `None` | v0.15.0 |
+| `request_timeout` | Timeout for HTTP requests | No | Positive integer or tuple of 2 integers | `None` | 
+| `max_retry_delay_seconds` | The maximum time between attempts to send failed events to the collector | No | int | 60s | 
+| `buffer_capacity` | The maximum capacity of the event buffer | No | int | `None` |
```

Comment:
Is it worth adding that despite a deafult value of None, it is technically 10,000 as noted below?